server/datastore/psql/maintenance_requests.go

```diff
@@ -0,0 +1,51 @@
+package psql
+
+import (
+	"time"
+
+	"github.com/a2fumn2022/gpmn-nsl/server/datastore"
+)
+
+// Import the necessary packages
+type rawTime []byte
+
+func (t rawTime) Time() (time.Time, error) {
+	return time.Parse("15:04:05", string(t))
+}
+
+func (db *DB) AllMaintenanceRequests() []datastore.MaintenanceRequest {
+
+	rows, err := db.connection.Query("select * from public.nsl_maintenance LIMIT $1", 777) //Number limits # of requests, can be anything
+	if err != nil {
+		// handle this error better than this
+		panic(err)
+	}
+
+	var D_Store []datastore.MaintenanceRequest
+	defer rows.Close()
+	for rows.Next() {
+
+		// Create a new MaintenanceRequest
+		var request2 datastore.MaintenanceRequest
```

Comment:
Let's call this `maintenance_request`. Not sure why there's the `2`.

Reply:
see above comment about underscores